packaging/standalone/src/main/distribution/shell-scripts/bin/Neo4j-Management.psd1

```diff
@@ -21,9 +21,9 @@
 
 
 @{
-ModuleVersion = '2.3.0'
+ModuleVersion = '3.0.0'
 
-GUID = 'dd9cad34-ad03-439b-b347-590625302c38'
+GUID = '2a3e34b4-5564-488e-aaf6-f2cba3f7f05d'
```

Comment:
Thanks. Doesn't that suggest that the GUID shouldn't change, because the module is logically the same (although a different version).